### PR TITLE
Submission bug

### PR DESCRIPTION
**Pull request by thecaitlinkinney**
@irislavie @vannguyen-vn @isabell-lee 

Fixes submission bug so that photo upload is not mandatory for submission 

Can you please review?

Trello: https://trello.com/c/I5nL5dzY

### CODE REVIEW

File: client/src/Components/RatingsAndReviews/AddReviewForm.jsx [hunk 1]
```diff
@@ -115,13 +116,11 @@ class AddReviewForm extends React.Component {
             'photos': photos,
             'characteristics': characteristics
           };
-
           this.props.submitReviewForm(dataBody);
-
-          this.closeModal();
-
+          console.log(dataBody);
```
Comment (isabell-lee) [inline client/src/Components/RatingsAndReviews/AddReviewForm.jsx:120]: Remove or comment out please

File: client/src/Components/RatingsAndReviews/RatingsAndReviews.jsx [hunk 2]
```diff
@@ -121,7 +121,7 @@ class RatingsAndReviews extends React.Component {
 
   submitReviewForm(body) {
     return axios.post('/reviews/', {params: body})
-      // .then((results) => console.log('AJAX POST RESULTS:', results.data, results))
+      .then((results) => console.log('AJAX POST RESULTS:', results.data, results))
```
Comment (isabell-lee) [inline client/src/Components/RatingsAndReviews/RatingsAndReviews.jsx:124]: Remove also?

Comment (thecaitlinkinney) [reply client/src/Components/RatingsAndReviews/RatingsAndReviews.jsx]: Done!

**Review by isabell-lee — APPROVED** [review]
Looks good!

**Pull request merged**